ios/sdk/src/AppInfo.m

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright @ 2017-present Atlassian Pty Ltd
```

Comment:
This module exports the app display name and version, which are not exposed by react-native, surprisingly. This allows us to customize messages with the app name, instead of hardcoding "Jitsi Meet".

In addition I added the version, since we may want to display it somewhere, eventually.